### DISCUSSION TITLE
Kivy - app money control

### DISCUSSION THREAD

**Discussion by joaosant05** [community/community#60735] [Mobile]
### Select Topic Area

Kivy, Mobile, Pandas

### Body

Good night!

I'm a new programer and need some tips

for my first project I wanna create a App to management my family monthly expenses... I have create one spreedsheet in excel ho make this function, but now i want make a app to facilite a use and filling to my family

in my first step i'm trying to use Kivy to create a apk, and now I trying to use pandas for management docs and database.. but i'm having some problems with this

anyway.. I'm open to tips to use pandas and if some know other libraries to make this project

thanks for atention ;)

**Comment by ossanna16** [comment]
Hi @joaosant05 ,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Kivy. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by joaosant05** [reply]
Hello @ossanna16,
 
 thank's to  tip, I start using github recently and I'm getting used to.. well, I go trying ask this in mobile category
 
 Thank's again and have a good day!

**Reply by ossanna16** [reply]
Happy to help @joaosant05 🙂
